queue-worker/main.go

```diff
@@ -210,9 +208,37 @@ func main() {
 			ackWait = ackWaitVal
 		}
 	}
+	var reconnectHandler nats.ConnHandler
+	var sub stan.Subscription
+	var sc stan.Conn
+	reconnectHandler = func(nc *nats.Conn) {
+		sc.Close()
+
+		sc, err = stan.Connect(clusterID, clientID, stan.NatsConn(nc))
```

Comment:
From here on, it repeats with what you do below for the initial setup. I would have made the `stan.Connect()` and creating of the queue sub a function (anonymous or not). You may, or may not need to have the print statements in that function, not sure what @alexellis preference would be. In the reconnect handler, you could however log that there was a reconnection.

Reply:
I think we should definitely log this. cc @vosmith

Reply:
@vosmith this is just debug information - not DRY - could we log any reconnects?

Reply:
Hey @alexellis , I can definitly log the reconnect here.  I apologize, I was focussed more on the first part of the comment about making the connection a separate function.  I forgot about the comment on logging.